algebra/src/fields/all_field_checks.sh

```diff
@@ -0,0 +1,86 @@
+echo "################ CHECKING ALL FIELD PARAMETERS ##################"
```

Comment:
@DDT92 Not related to this file, but the check_field_parameters.sage: 
Please optimize the test for the root of unity.

Reply:
Done.